shared/src/libs/packages/file/libs/enums/form-data-key.enum.ts

```diff
@@ -1,5 +1,6 @@
 const FormDataKey = {
   FILE: 'file',
+  PROJECT_ID: 'project_id',
```

Comment:
```suggestion
```
take it from URL

`/projects/:id/avatar`

Reply:
Not sure if it was resolved

Reply:
remove it